packages/webapp/src/util/index.js

```diff
@@ -131,3 +132,13 @@ export const getObjectInnerValues = (data) => {
     return { ...reduced, [k]: data[k] };
   }, {});
 };
+
+export const truncateText = (text, length) => {
+  if (text) {
+    if (text.length > length) {
+      return text.slice(0, 22) + '...';
+    }
+    return text;
+  }
+  return '';
```

Comment:
@MwayaB 

Possible one liner but up to you: 

`return text?.length > length ? text.slice(0, 22) + '...' : text ?? '';`

Reply:
That's beautiful, but not very readable to someone new to javascript. I think i'll leave it as is